### PR TITLE
Change `customize_loss_grad` to `use_default_grad_scale`.

### LINKED ISSUE #10219: Refine argument naming for ParallelExecutor
Seems `customize_loss_grad` is not very self-explanatory and there's is no doc for this argument. I propose to change it to a more clear name like `use_default_grad_scale` and the default value is `true`. @reyoung @panyx0718 Need your opinion. Thanks. 

https://github.com/PaddlePaddle/Paddle/blob/bfafcbeefbdddd581608b2a5bd3559a2175503e2/python/paddle/fluid/parallel_executor.py#L33

### PR DESCRIPTION
**Pull request by pkuyym**
Resolves #10219

### CODE REVIEW

File: python/paddle/fluid/parallel_executor.py [hunk 1]
```diff
@@ -46,6 +46,10 @@ def __init__(self,
                 improve performance in some cases, defalut False.
             share_vars_from(ParallelExecutor, default None): If provied,
                 it will share variables from the specified ParallelExecutor.
+            use_default_grad_scale(bool, default True): If set True, a default
+                scale value equal to `1./device_count` would be multiplied to
+                the gradients. Otherwise, a customized scale value should be
```
Comment (panyx0718) [inline python/paddle/fluid/parallel_executor.py]: to gradients of each device? and then aggregated?

File: python/paddle/fluid/parallel_executor.py [hunk 2]
```diff
@@ -46,6 +46,10 @@ def __init__(self,
...
+                feeded to the network.
```
Comment (panyx0718) [inline python/paddle/fluid/parallel_executor.py]: feeded->fed?

Comment (pkuyym) [reply python/paddle/fluid/parallel_executor.py]: Done.

File: python/paddle/fluid/parallel_executor.py (hunk 1 above)
Comment (pkuyym) [reply python/paddle/fluid/parallel_executor.py]: Thanks, followed the comment.

**Pull request merged**